decryptor/mysql/response_proxy.go

```diff
@@ -265,7 +269,7 @@ func (handler *MysqlHandler) ClientToDbConnector(errCh chan<- error) {
 				}
 				continue
 			}
-			clientLog.WithField("sql", sqlQuery).Debugln("Com_query")
+			clientLog.WithField("sql", queryWithHiddenValues).Debugln("com_query")
```

Comment:
`Com_query` (instead lower cased `com_query`) was used because we use capitalized log messages :)